ansible_navigator/actions/open_file.py

```diff
@@ -48,66 +43,21 @@ def __init__(self, args):
         self._args = args
         self._logger = logging.getLogger(__name__)
 
-    def _content(self, content: Content, match: Match) -> Tuple[Union[str, None], str, Any]:
-        filename = None
-        line_number = "0"
-        obj = None
-        self._logger.debug("content is showing")
-        something = match.groupdict()["something"]
-        if something:
-            self._logger.debug("asked to open something")
-            if something.startswith("{{"):
-                self._logger.debug("something appears to be a template: %s", something)
-                templated = templar(something, content.showing)
-                if isinstance(templated, str):
-                    parts = templated.rsplit(":", 1)
-                    if os.path.isfile(parts[0]):
-                        filename = parts[0]
-                        line_number = parts[1:][0] if parts[1:] else line_number
-                        self._logger.debug(
-                            "template interaction in valid filename %s:%s", filename, line_number
-                        )
-                    else:
-                        self._logger.debug("template not a valid filename, open showing")
-                        obj = templated
-                else:
-                    self._logger.debug("template not a string, open showing")
-                    obj = templated
-            else:
-                parts = something.rsplit(":", 1)
-                if os.path.isfile(parts[0]):
-                    filename = parts[0]
-                    line_number = parts[1:][0] if parts[1:] else line_number
-                    self._logger.debug(
-                        "something not a template, but is a valid filename %s:%s",
-                        filename,
-                        line_number,
-                    )
-                else:
-                    self._logger.debug("something just a plain string")
-                    obj = something
-        else:
-            self._logger.debug("something not provided")
-            obj = content.showing
-        return filename, line_number, obj
-
     @staticmethod
     def _remove_dbl_un(string):
```

Comment:
The method seems to be repeated at 2 places, probably can be move in utils file

Reply:
good idea, will do